### ISSUE TITLE
Replace sum of average losses per batch with average over entire validation set

### ISSUE THREAD

**Issue by LeonHvastja** [davidnabergoj/normalizing-flows#7]
In the linked comment, validation loss is computed as the sum over batch losses, each batch loss being averaged. It would be better if this was the average over all validation data points.

https://github.com/davidnabergoj/normalizing-flows/blob/7b83bd0411242f0931452c10620fed987ca1bf1c/normalizing_flows/flows.py#L232

**Comment by davidnabergoj** [comment]
Thanks for posting the issue, this has been fixed in 23d319e and will be merged into the master branch soon.
